### PR TITLE
Mass update to framework

### PR DESCRIPTION
**Pull request by januszgrze**
I am proposing that we update our risk reviews per a new framework in this PR. It is currently in draft.

### CODE REVIEW

File: content/layers/bevm.json [hunk 1]
```diff
@@ -23,14 +23,14 @@
         "https://github.com/btclayer2/BEVM",
         "https://twitter.com/BTCLayer2"
     ],
-    "description": "BEVM is an EVM-compatible sidechain, built on substrate, designed for Bitcoin. On the BEVM chain, the BTC is held in custody by a federated multi-signature address. Its goal is to make it easier for Bitcoin to be seamlessly integrated into the Ethereum ecosystem.",
+    "description": "BEVM is an EVM-compatible sidechainmbuilt on substrate. On the BEVM chain, the BTC is held in custody by a federated signer set.",
```
Comment (veronikakuett) [inline content/layers/bevm.json:26]: typo: sidechainmbuilt

File: content/layers/bevm.json [hunk 2]
```diff
@@ -83,7 +83,7 @@
             "content": [
                 {
                     "title": "Users trust the trustees on BEVM chain to process their withdrawals",
-                    "content": "Upon the initiation of a withdrawal transaction,  trustees sign the withdrawal and if the number of signatures reaches the multi-signature requirement, the transaction will be broadcast to the Bitcoin Network."
+                    "content": "Upon the initiation of a withdrawal transaction on the BEVM chain, trustees will be notified of the withdrawal request, sign the withdrawal, and if the number of signatures reaches the multi-signature requirement, the transaction will be broadcasted to the bitcoin Network."
```
Comment (veronikakuett) [inline content/layers/bevm.json:86]: if "bitcoin" is spelled with lower letters, then also "Network" --> network

File: content/layers/bitlayer.json [hunk 3]
```diff
@@ -41,18 +41,18 @@
             "content": "Bitlayer does not currently use Bitcoin for data availability. The data availability requirement is currently fufilled by Bitlayer full nodes. The Bitlayer node software is open source and anyone can run a node and validate the state of Bitlayer."
         },
         {
-            "category": "Network Operators",
+            "category": "Liveness & Reorg Resistance",
```
Comment (veronikakuett) [inline content/layers/bitlayer.json:44]: New category term: "liveness & finality"

**Comment by januszgrze** [comment]
@RedVelvetZip @veronikakuett - please review

File: content/layers/internetcomputer.json [hunk 4]
```diff
@@ -27,11 +27,11 @@
...
+            "tier": "Medium",
+            "title": "Funds are custodied by signer set selected by ICP consensus",
+            "content": "Users who deposit funds into ckBTC trust a set of operators, who are elected via ICP governance, with the custody of their bitcoin. The operators of the ‘pzp6e…’ subnet manage the “ckBTC” smart contract module, which is responsible for minting, custodying and burning bitcoin-backed tokens on the ICP sidechain.\n\nThis smart contract is a part of a subnet with 28 node operators. These operators have undergone a KYB process to ICP governance and are publicly known."
```
Comment (RedVelvetZip) [inline content/layers/internetcomputer.json:34]: @januszgrze confirm KYB

**Review by RedVelvetZip — APPROVED** [review]
ack

**Pull request merged**